scripts/pylint_extensions.py

```diff
@@ -757,6 +757,33 @@ def process_module(self, node):
                         'backslash-continuation', line=line_num + 1)
 
 
+class FunctionArgsOrderChecker(checkers.BaseChecker):
+    """Checker for order of arguments. It checks
```

Comment:
I think we can have more characters in this line.

Reply:
Also, this docstring is wrongly phrased. Change it to:
`Custom pylint checker which checks the order of arguments in function definition.`

Reply:
Okay will change it .